flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "A js2nix flake";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs?rev=13e0d337037b3f59eccbbdf3bc1fe7b1e55c93fd";
+  };
+
+  outputs = { self, nixpkgs }:
+    let
+      supportedSystems = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
+      forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
+    in
+    rec {
+      packages = forAllSystems (system:
+        let
+          pkgs' = nixpkgs.legacyPackages.${system};
+          js2nix = pkgs'.callPackage ./. { };
+        in
+        {
+          default = js2nix;
```

Comment:
I'd sort of expect this to be in a `lib.${system}` namespace. "Semantics over impl-detail"

Reply:
potentially even make that function curriable with  an instance of `pkgs`:

```nix
lib.js2nix pkgs {
      package-json = ./package.json;
      yarn-lock = ./yarn.lock;
}
```

I always feel immediately at home when there is special consideration put to the handling of `pkgs` instance. :smile:

Reply:
Thanks for the feedback around the `lib` namespace @blaggacao, that's good point! I'll incorporate it in coming commit.